doc/interop-test-descriptions.md

```diff
@@ -1078,6 +1095,15 @@ included in that latency measurement (channel teardown semantics differ widely
 between languages). This latency measurement should also be the value that is
 logged and recorded in the latency histogram.
 
+* `createNewChannel` Function: In channel_soak, the `createNewChannel` function 
```

Comment:
nit: I think no need to document `createNewChannel` since it's just an implementation detail.

Semantics are implied already by "Similar to rpc_soak, but this time each RPC is performed on a new channel. The
channel is created just before each RPC and is destroyed just after."